### PR TITLE
Visitor must fill in location when writing article

### PR DESCRIPTION
**Pull request by stefankarlberg**
### Changes we propose in this PR:
* Visitor can now create an article and fill in country and city.
* Country and city is displayed in the full article view.
* Country and city is displayed in the list of articles in the landing page. 

 ### What have we learned:
* Better understanding of form handling events.
* Installing and configuring packages of NPM.
* How many countries there are in the world.

Screenshot 2019-05-30 at 13 24 36

### CODE REVIEW

File: src/Components/WriteArticle.jsx [hunk 1]
```diff
@@ -2,6 +2,7 @@ import React, { Component } from 'react'
 import { Form, Container, Button, Message, Dropdown } from 'semantic-ui-react'
 import { Redirect } from 'react-router-dom'
 import axios from 'axios'
+import { COUNTRY_OPTIONS } from './countriesData.js'
```
Comment (leiter007) [inline src/Components/WriteArticle.jsx:5]: Just out of interest, where are you taking these country options from?

**Review by leiter007 — COMMENTED** [review]
Looks good I think!

File: src/Components/WriteArticle.jsx (hunk 1 above)
Comment (raptorf1) [reply src/Components/WriteArticle.jsx:5]: We installed an NPM package that actually creates a js file with all the countries and their 2 letter abbreviations. You can see the file inside the Components folder. The file is ready to be used as an options file of the `Dropdown` semantic component.

File: cypress/fixtures/categories_list.json [hunk 2]
```diff
@@ -0,0 +1,54 @@
+[
+  {
+    "id": 1,
+    "name": "Politics"
+  },
+  {
+    "id": 2,
+    "name": "Opinion"
+  },
+  {
```
Comment (oliverochman) [inline cypress/fixtures/categories_list.json:10]: dont we already have a fixture file for the `get` request for categories

Comment (raptorf1) [reply cypress/fixtures/categories_list.json:10]: done ;)

File: src/Components/WriteArticle.jsx (hunk 1 above)
Comment (leiter007) [reply src/Components/WriteArticle.jsx:5]: > We installed an NPM package that actually creates a js file with all the countries and their 2 letter abbreviations. You can see the file inside the Components folder. The file is ready to be used as an options file of the `Dropdown` semantic component.

Nice! Good to know :)

**Pull request merged**